test/zdtm.py

```diff
@@ -1032,6 +1036,11 @@ def run(action,
 
         return ret
 
+    @staticmethod
+    def signal_exit(ret):
+        # TODO
```

Comment:
TODO should contain a comment. Why can't we implement this method for the rpc?

Reply:
@avagin AFAIU, `return ret < 0` can't be used here as the `wait()` function of the `criu_rpc` class sets `ret` as negative even in the case of non-Signal exits, as given in the `CRIUExceptionExternal` class in the `test/pycriu/criu.py` file. This would lead to false positives.

Reply:
@avagin An instance of the `CRIUExceptionExternal` class always has an *errno* attribute which is the error code returned by the process using the `errno` module in Python. One of these error codes is `EINTR: Interrupted system call` which is raised when a system call is interrupted by a signal. AFAIU, this should work, I would be glad to here any better solutions or alternatives :)

Reply:
Have you tested your change? If the answer is yes, pls describe the process.

Reply:
No, I haven't been able to successfully run the `zdtm` tests.